frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.scss

```
@@ -4,3 +4,7 @@
   justify-content: space-between;
   align-items: center;
 }
+
+.all-revisions-link {
```

namespace it with odc
```suggestion
.odc-revisions-link {
```

---

done